src/math_utils.rs

```diff
@@ -3,6 +3,15 @@ use num_bigint::BigInt;
 use num_integer::Integer;
 use num_traits::{abs, FromPrimitive};
 
+/// Returns the lift of the given field element, val, as an integer in the range (-prime/2, prime/2).
+pub fn as_int(val: &BigInt, prime: &BigInt) -> BigInt {
```

Comment:
Why is it called `as_int`?

Reply:
Thats the name they gave it in math_utils.py in the common library

Reply:
This function is used by various hints, so its better to keep the same name

Reply:
Works for me.